week3/homework/README.md

```diff
@@ -9,19 +9,19 @@ Add four more actions:
 
 ### `readTodo` (`GET /todos/:id`)
 
-  Get a single to-do with ID `:id`
+Get a single to-do with ID `:id`
 
 ### `clearTodos` (`DELETE /todos`)
 
-  Clears the list of to-dos
+Clears the list of to-dos
 
-### `markAsDone` (`POST /todos/:id/done`)
+### `markAsDone` (PATCH /todos/:id)
 
-  Sets the `done` flag of a single to-do to `true`
+Sets the done flag of a single to-do to true
```

Comment:
We should put back the back-ticks (`) around the endpoint, done and true , same below.
Also the endpoint should still have a done at the end maybe? or if there is a body to the request it should be specified.

Reply:
Corrected backticks and end point

Reply:
I think the end point should be 
PATCH /todos/:id

This makes it more general. It can handle both MarkAsDone and MarkAsUndone as well as changing other properties. It is all specified in the body. The model itself may have specific methods for Done, Undone, etc.